tooling/debugger/src/repl.rs

```diff
@@ -32,6 +32,9 @@ pub struct ReplDebugger<'a, B: BlackBoxFunctionSolver<FieldElement>> {
 
     // Brillig functions referenced from the ACIR circuits above
     unconstrained_functions: &'a [BrilligBytecode<FieldElement>],
+
+    // whether to print the source without highlighting / pretty-printing
+    raw_source_printing: bool,
```

Comment:
It seems this option also removes line numbers, right? I don't know if that's intentional, but it should probably be documented too.

Reply:
Yes, it intentionally removes line numbers. I've updated the comment.